pageobjects/MainPage.js

```diff
@@ -53,16 +58,26 @@ class MainPage extends BasePage {
 
   async fillTheField(fieldName, text) {
     const element = this.getFieldByName(fieldName);
-    console.log(element);
     await Waiters.waitElementIsDisplayed(element);
     await ElementAction.setValueInField(element, text);
     await ElementAction.click(this.firstLocation(text));
+    await ElementAction.click(element);
+    await ElementAction.sendKeys('\uE007');
   }
 
   async titleContains(title) {
     await expect(browser).toHaveTitleContaining(title);
   }
-}
 
+  async chooseTheDataFromCalendar(dataOne, dataTwo) {
+    const dataFrom = this.dataInCalendar(dataOne);
```

Comment:
change parameters' names

Reply:
change function name to `selectDatesFromCalender`

Reply:
I removed this code too